justfile

```
@@ -92,9 +94,13 @@ _build-release crate:
 _test-release crate:
     just _build-release {{crate}}
     (cd "{{crate}}" && cargo test --release)
+_test-release-arm crate:
+    (cd "{{crate}}" && cargo test --release --target={{justfile_directory()+"/armv4t-none-eabi.json"}})
```

```suggestion
    (cd "{{crate}}" && cargo test --release --target="{{justfile_directory()+"/armv4t-none-eabi.json"}}")
```

do we want extra quotes around here so that it works with folders with spaces in their name?